R/method-get-tree.R

```diff
@@ -65,3 +65,24 @@ setMethod("get.tree", signature(object="phylo"),
           function(object, ...) {
               return(object)
           })
+
+##' @rdname get.tree-methods
+##' @exportMethod get.tree
+setMethod("get.tree", signature(object="data.frame"),
+          function(object, ...) {
+              edge <- object[, c("parent", "node")]
+              edge <- edge[edge[,1] != 0 & edge[,1] != edge[,2], ]
+              edge.length <- object[, "branch.length"]
```

Comment:
i <- which(edge[,1] != 0 & edge[,1] != edge[,2])
+              edge <- edge[i, ]
+              edge.length <- object[i, "branch.length"]

otherwise edge.length will longer than nrow(edge)